app/javascript/dashboard/components/ChatList.vue

```diff
@@ -347,18 +348,39 @@ export default {
         this.currentPageFilterKey
       );
     },
+    activeAssigneeTabCount() {
+      const { activeAssigneeTab } = this;
+      const count = this.assigneeTabItems.find(
+        item => item.key === activeAssigneeTab
+      ).count;
+      return count;
+    },
     conversationFilters() {
       return {
         inboxId: this.conversationInbox ? this.conversationInbox : undefined,
         assigneeType: this.activeAssigneeTab,
         status: this.activeStatus,
-        page: this.currentPage + 1,
+        page: this.conversationListPagination,
         labels: this.label ? [this.label] : undefined,
         teamId: this.teamId || undefined,
         conversationType: this.conversationType || undefined,
         folders: this.hasActiveFolders ? this.savedFoldersValue : undefined,
       };
     },
+    conversationListPagination() {
+      const conversationsPerPage = 25;
+      if (!this.hasAppliedFiltersOrActiveFolders && this.chatList !== []) {
```

Comment:
We can move the conditions inside as variables. So we could combine the two if's into a single if condition as in both cases we are returning `return this.currentPage + 1;`.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6899/commits/f0be840e4db6b3b149d874105749ca53925e0aca